build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.build-logic.kotlin-dsl-gradle-plugin.gradle.kts

```diff
@@ -48,6 +49,11 @@ ktlint {
     }
 }
 
+tasks.named<KtlintCheckTask>("ktlintKotlinScriptCheck") {
```

Comment:
Since you applied the plugin in the plugins block, you should use the generated accessors:

`tasks.ktlintKotlinScriptCheck {}`

Reply:
Done.